src/pudl/metadata/codes.py

```diff
@@ -1231,4 +1231,256 @@
+                (
+                    "SOCO",
+                    "southern_company_services",
+                    "Southern Company Services, Inc. - Trans",
+                ),
+                ("SPA", "southwestern_power", "Southwestern Power Administration"),
+                (
+                    "SPRM",
+                    "city_utilities_of_springfield",
+                    "City Utilities Of Springfield, MO",
+                ),
+                (
+                    "SPS",
+                    "southwestern_public_service",
+                    "Southwestern Public Service Co. (Xcel Energy)",
+                ),
+                ("SRP", "salt_river_project", "Salt River Project"),
+                ("SWPP", "southwest_power_pool", "Southwest Power Pool"),
+                ("TAL", "city_of_tallahassee", "City of Tallahassee"),
+                ("TEC", "tampa_electric", "Tampa Electric Company"),
+                ("TEPC", "tuscon_electric_power", "Tucson Electric Power Company"),
+                ("TIDC", "turlock_irrigation_district", "Turlock Irrigation District"),
+                (
+                    "TPWR",
+                    "city_of_tacoma",
+                    "City of Tacoma, Department of Public Utilities, Light Division",
+                ),
+                ("TVA", "tennessee_valley_authority", "Tennessee Valley Authority"),
+                (
+                    "WACM",
+                    "western_area_power_mountain",
+                    "Western Area Power Administration - Rocky Mountain Region",
+                ),
+                (
+                    "WALC",
+                    "western_area_power_southwest",
+                    "Western Area Power Administration - Desert Southwest Region",
+                ),
+                (
+                    "WAUE",
+                    "western_area_power_east",
+                    "Western Area Power Administration - Upper Great Plains East",
+                ),
+                (
+                    "WAUW",
+                    "wester_area_power_west",
+                    "Western Area Power Administration UGP West",
+                ),
+                (
+                    "WFEC",
+                    "western_farmers_electric_coop",
+                    "Western Farmers Electric Cooperative",
+                ),
+                ("WR", "westar_energy", "Westar Energy"),
+                ("WWA", "naturener_wind", "NaturEner Wind Watch, LLC"),
+                (
+                    "YAD",
+                    "alcoa_power_yadkin",
+                    "Alcoa Power Generating, Inc. - Yadkin Division",
+                ),
+                ("GLHB", "GridLiance", "GridLiance (GLHB)"),
+            ],
+        ).convert_dtypes(),
+        "code_fixes": {
+            "NVE": "NEVP",
+            "IS": "ISNE",
+            "PS": "PSCO",
+            "TIC": "TIDC",
+            "TID": "TIDC",
+            "CA": "CISO",
+            # KACY > SWPP
```

Comment:
What does this comment about KACY & SWPP mean?

Reply:
this was a leftover from my exploration of the plants that switched BA's over time. LOTS of plants switched from small (rarely reported) BA's to SWPP. Which from my understanding of SWPP that seems accurate. I will remove this comment!